pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -881,6 +881,7 @@ public void unloadNamespace(@Suspended final AsyncResponse asyncResponse, @PathP
     @ApiOperation(hidden = true, value = "Unload a namespace bundle")
     @ApiResponses(value = {
             @ApiResponse(code = 307, message = "Current broker doesn't serve the namespace"),
+            @ApiResponse(code = 404, message = "Namespace doesn't exist"),
```

Comment:
Nit: I think we probably want all of these to be like this:
```suggestion
            @ApiResponse(code = 404, message = "Namespace does not exist"),
```